### PR TITLE
Feature/directory structure

### PR DESCRIPTION
**Pull request by jcwenger**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,8 +0,0 @@
-[submodule "aasdk"]
```
Comment (rsjudka) [inline .gitmodules:1]: bless no more submodules 🙏

File: CMakeLists.txt [hunk 2]
```diff
@@ -92,18 +99,12 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}
...
-add_executable(ia ${source_files})
 
-add_dependencies(auto aasdk)
 target_link_libraries(ia
                       aasdk_proto
                       aasdk
-                      auto
-                      ${Boost_LIBRARIES}
+                      openauto
+		      ${Boost_LIBRARIES}
```
Comment (rsjudka) [inline CMakeLists.txt:107]: should be spaces here not tabs

**Comment by rsjudka** [comment]
looks like some of the spaces turned to tabs.. could you clean those up?

File: src/app/widgets/color_label.cpp [hunk 3]
```diff
@@ -46,4 +46,4 @@ void ColorLabel::update(QString color)
     QPixmap block(this->icon->pixmap()->size());
     block.fill(this->theme->get_color(color));
     this->icon->setPixmap(block);
-}
\ No newline at end of file
+}
```
Comment (rsjudka) [inline src/app/widgets/color_label.cpp:49]: add the newline back

File: CMakeLists.txt [hunk 4]
```diff
@@ -71,7 +68,17 @@ if(GST_BUILD)
 endif(GST_BUILD)
 message(INFO "${GST_LIBRARIES}")
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}
+link_directories(${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
+file(GLOB_RECURSE source_files ${sources_directory}/*.cpp ${include_directory}/*.hpp ${resources_directory}/*.qrc)
+
+add_executable(ia ${source_files})
```
Comment (rsjudka) [inline CMakeLists.txt:75]: this should change to `dash`? (also the references below it)

when I built it, it created a binary called ia

File: src/app/widgets/color_label.cpp (hunk 3 above)
Comment (jcwenger) [reply src/app/widgets/color_label.cpp:49]: Not removed in my commit, but newline restored.

File: CMakeLists.txt (hunk 4 above)
Comment (jcwenger) [reply CMakeLists.txt:75]: Yes, but not in this set of commits.  There's more work to be done here like killing GLOB, scrubbing include directory and link lib stuff.  I think we're including a lot of stuff that's not truly necessary.

**Pull request merged**